gating/update_dependencies/get-rpc_release.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# Copyright 2014-2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import yaml
+
+# Read the release file path from an environment variable
+if len(sys.argv) > 1:
+    release_file = sys.argv[1]
+else:
+    release_file = os.environ['RELEASE_FILE']
+
+# Read the RPC release series name from an environment variable
+if len(sys.argv) > 2:
+    release_series = sys.argv[2]
+else:
+    release_series = os.environ['RPC_PRODUCT_RELEASE']
+
```

Comment:
would using argparse or similar not make this less brittle?

Reply:
Perhaps, but that can be evolved from here. Given the timelines I went with quick and simple.